### PR TITLE
Fix balance failed

### PR DESCRIPTION
**Pull request by laura-ding**
https://discuss.nebula-graph.com.cn/t/topic/1449

### CODE REVIEW

**Review by critical27 — DISMISSED** [review]
Gooood catch, thanks taking care of it!

File: src/meta/processors/admin/Balancer.cpp [hunk 1]
```diff
@@ -471,9 +471,12 @@ cpp2::ErrorCode Balancer::leaderBalance() {
                 }
             }
         }).wait();
-        LOG(ERROR) << failed << " partiton failed to transfer leader";
         inLeaderBalance_ = false;
-        return cpp2::ErrorCode::E_BALANCER_FAILURE;
+        if (failed > 0) {
```
Comment (panda-sheep) [inline src/meta/processors/admin/Balancer.cpp:475]: Good catch！

**Review by panda-sheep — APPROVED** [review]
Excellent！👏

File: src/meta/processors/admin/Balancer.cpp (hunk 1 above)
Comment (critical27) [reply src/meta/processors/admin/Balancer.cpp:475]: Hold on, @laura-ding, this place is intended to do that, we don't expect all transfer leader is successful, no need to return failure.

**Comment by critical27** [comment]
Hold on, don't merge plz

**Review by critical27 — APPROVED** [review]
Well done

**Review by bright-starry-sky — APPROVED** [review]
Well done.

**Pull request merged**